<launch>
  
  <!-- Run the map server > 
  <node name="map_server" pkg="map_server" type="map_server" args="$(find my_map_package)/my_map.pgm my_map_resolution"/-->
  
  <!-- group ns = "planner"-->  
  
    <remap from="map" to="/projected_map"/>
    
    <remap from="scan" to="/front/scan"/>
    <remap from="odom" to="/odometry/filtered"/>
      

    <!-- Run AMCL, don't use, only for use in static premade maps > 
    <include file="$(find amcl)/examples/amcl_diff.launch" /-->

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="log">
      
      <rosparam file="$(find jackal_2dnav)/params/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
      <rosparam file="$(find jackal_2dnav)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
      <rosparam file="$(find jackal_2dnav)/params/local_costmap_params.yaml" command="load" />
      <rosparam file="$(find jackal_2dnav)/params/global_costmap_params.yaml" command="load" /> 
    
      <rosparam file="$(find jackal_2dnav)/params/base_local_planner_params.yaml" command="load" />
    </node>
    
  <!--/group-->

</launch> 
